jnagmp/src/test/java/com/squareup/jnagmp/GmpTest.java

```diff
@@ -364,8 +334,8 @@ private void testEvenExamples() {
   }
 
   @Test public void testSmallExhaustiveSecure() {
-    for (int base = 10; base >= 0; --base) {
-      for (int exp = 10; exp >= 0; --exp) {
+    for (int base = 10; base >= -10; --base) {
+      for (int exp = 10; exp >= -10; --exp) {
```

Comment:
@JakeWharton those cases are tested here and 311.  We're testing that the GMP stuff either returns the same result or else throws the same exception type as Java BigInteger